<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterBorneStationnementInterface</class>
 <widget class="QDialog" name="AjouterBorneStationnementInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter Borne de Stationnement </string>
  </property>
  <widget class="QLabel" name="lbIdentifiant">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Identifiant de la borne&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="txtIdentifiant">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbDirection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Direction&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbNomTopo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Nom topographique&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLongitude">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Longitude&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLatitude">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Latitude&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboDirection">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nord</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sud</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Est</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ouest</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>null</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDistance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Distance mesuree&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbNumBorne">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Num√©ro de la borne&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSegment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Segment de rue&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbCote">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Cote de la rue&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNomTopo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="txtLongitude">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="maximum">
    <double>1000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="txtLatitude">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="maximum">
    <double>1000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="comboType">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>stationnement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>paiement</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="txtDistance">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="txtSegment">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="txtNumBorne">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboCote">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nord</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sud</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Est</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ouest</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>null</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnSoumettre">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Soumettre</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnSoumettre</sender>
   <signal>clicked()</signal>
   <receiver>AjouterBorneStationnementInterface</receiver>
   <slot>validerChamps()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerChamps()</slot>
 </slots>
</ui>
